pkg/result/ignore.go

```diff
@@ -184,7 +184,8 @@ func (c *IgnoreConfig) MatchLicense(licenseID, filePath string) *IgnoreFinding {
 func ParseIgnoreFile(ctx context.Context, ignoreFile string) (IgnoreConfig, error) {
 	var conf IgnoreConfig
 	if _, err := os.Stat(ignoreFile); errors.Is(err, fs.ErrNotExist) {
-		// .trivyignore doesn't necessarily exist
+		// .trivyignore doesn't necessarily exist or maybe empty
```

Comment:
Empty files are not handled here now.

```suggestion
		// .trivyignore doesn't necessarily exist
```